cpp/velox/compute/VeloxBackend.cc

```diff
@@ -239,6 +239,7 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   registerConnector(hiveConnector);
   velox::parquet::registerParquetReaderFactory();
   velox::dwrf::registerDwrfReaderFactory();
+  velox::dwrf::registerOrcReaderFactory();
```

Comment:
PR #3445 is going to remove the registration for Parquet and DWRF reader. I'm wondering if the registration for ORC reader can also be done in Velox, then this change can also be removed.

Reply:
OK, Adjusted this in PR [417](https://github.com/oap-project/velox/pull/417)
After that merge, I can submit a new PR base on main branch